contracts/extensions/VotingReputation/VotingReputationMisalignedRecovery.sol

```diff
@@ -0,0 +1,225 @@
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.7.3;
+pragma experimental ABIEncoderV2;
+
+import "./../../colonyNetwork/IColonyNetwork.sol";
+import "./../../colony/ColonyRoles.sol";
+import "./../../common/BasicMetaTransaction.sol";
+import "./../../common/ERC20Extended.sol";
+import "./../../patriciaTree/PatriciaTreeProofs.sol";
+import "./../../tokenLocking/ITokenLocking.sol";
+import "./../ColonyExtension.sol";
+import "./../../../lib/dappsys/math.sol";
+import "./VotingReputationDataTypes.sol";
+
+contract VotingReputationMisalignedRecovery is PatriciaTreeProofs, DSMath, DSAuth, VotingReputationDataTypes {
+
+  // THIS FILE IS DELIBERATELY WRONG. IF YOU'RE EDITING THIS FILE, AND YOU'VE NOT BEEN EXPLICITLY
+  // TOLD TO DO SO, LEAVE NOW. THERE BE DRAGONS HERE.
+
+  // Constants
+  uint256 constant UINT128_MAX = 2**128 - 1;
+
+  uint256 constant NAY = 0;
+  uint256 constant YAY = 1;
+
+  address resolver; // Align storage with EtherRouter
+
+  IColony colony;
+  bool deprecated;
+
+  ExtensionState state;
+
+  IColonyNetwork colonyNetwork;
+  ITokenLocking tokenLocking;
+  address token;
+
+  // All `Fraction` variables are stored as WADs i.e. fixed-point numbers with 18 digits after the radix. So
+  // 1 WAD = 10**18, which is interpreted as 1.
+
+  uint256 totalStakeFraction; // Fraction of the domain's reputation needed to stake on each side in order to go to a motion.
+  // This can be set to a maximum of 0.5.
+  uint256 voterRewardFraction; // Fraction of staked tokens paid out to voters as rewards. This will be paid from the staked
+  // tokens of the losing side. This can be set to a maximum of 0.5.
+
+  uint256 userMinStakeFraction; // Minimum stake as fraction of required stake. 1 means a single user will be required to
+  // provide the whole stake on each side, which may not be possible depending on totalStakeFraction and the distribution of
+  // reputation in a domain.
+  uint256 maxVoteFraction; // Fraction of total domain reputation that needs to commit votes before closing to further votes.
+  // Setting this to anything other than 1 will mean it is likely not all those eligible to vote will be able to do so.
+
+  // All `Period` variables are second-denominated
+
+  uint256 stakePeriod; // Length of time for staking
+  uint256 submitPeriod; // Length of time for submitting votes
+  uint256 revealPeriod; // Length of time for revealing votes
+  uint256 escalationPeriod; // Length of time for escalating after a vote
+
+  // Here we deliberately recreate the misalignment in the storage slots, so solidity can correctly
```

Comment:
Except the comments to do with the misaligned storage

Reply:
I would be inclined also to re-name the unused variables (unused in the two functions we implement) with dummy names to further emphasize that they are not used for anything.